### PR TITLE
Add note about autoprefixer to documentation

### PR DESCRIPTION
**Pull request by anselmbradford**
## Additions

- Adds note about autoprefixer to docs needed by https://github.com/cfpb/capital-framework/pull/351.

## Review

- @jimmynotjim 
- @Scotchester

### CODE REVIEW

File: getting-started/index.md [hunk 1]
```diff
@@ -74,6 +74,10 @@ just remember to import them in `main.less` using the correct path.
 If you don't want to use the generator, you can just download the source files and
 import them into your project.
 
+> NOTE: Be sure to run the less files through
```
Comment (Scotchester) [inline getting-started/index.md]: Capitalize "Less".

**Comment by jimmynotjim** [comment]
👍

File: getting-started/index.md (hunk 1 above)
Comment (anselmbradford) [inline getting-started/index.md]: Fixed in https://github.com/cfpb/capital-framework/pull/355/commits/62add6da5651839d9c23ec441ba77f4e36976804

**Comment by anselmbradford** [comment]
code review commits in the history 😱  😉

**Comment by Scotchester** [comment]
Oops ;)

**Pull request merged**